### ISSUE TITLE
Help Building from Source

### ISSUE THREAD

**Issue by AuthenticAMD** [blue-yonder/turbodbc#232]
Hello,

Trying to build turbodbc from source (no pip), but having the error:

In file included from src/turbodbc_python/python_result_set.cpp:1:0:
include/turbodbc_python/python_result_set.h:4:31: fatal error: pybind11/pybind11.h: No such file or directory

Sorry there is not as much detail because I have to dictate error messages and output by hand as the system is on a standalone network.

The following is installed:

RHEL 7.6
Python 3.7
Pybind11 2.4.3
libboost-all-dev
unixodbc-dev
python-dev

Any thoughts on what I can do next?

**Comment by xhochy** [comment]
This is possibly happening because you have not downloaded the git submodules. Please run in your local checkout:

```
git submodules init
git submodules update
```

**Comment by AuthenticAMD** [comment]
I'm assuming that requires internet access to this project. If so, I'm on a stand alone network so that make it challenging. Are the submodules in the latest tar.gz?

**Comment by xhochy** [comment]
No, they aren't. I would suggest to clone with submodules where you have internet and copy over all files then to your standalone network. Alternatively, a PR to use pre-installed pybind11 is always welcomed.
